store/src/java/com/zimbra/cs/service/admin/ModifyServer.java

```diff
@@ -58,6 +67,8 @@ public Element handle(Element request, Map<String, Object> context) throws Servi
         }
         checkRight(zsc, context, server, attrs);
 
+        startOrStopPostSRSd(server, attrs);
```

Comment:
is this going to be called with every modifyServer call?

Reply:
@silentsakky, I think the same this code should not be here, as zmcontrol and all other service's stop/start functionality is written in zm-core-utils repo, which is in Perl script. This logic should also be written in that repo, also would like to confirm the same with @desouzas . 
@desouzas : Could you please provide your inputs here? Thanks..!!